### ISSUE TITLE
Clone from Visitors & Migrants rather than CareConnect

### ISSUE THREAD

**Issue by agreenhalgh** [nhsconnect/FHIR-MEDICATIONS-API#3]
The Visitors & Migrants DMS is much closer to the product here than CareConnect so would provide a much better starting point leaving us with much less to change/get rid of. Could we retain the small changes in to date to date and clone/rebase V&M instead.

**Comment by ahatherly** [comment]
I'm guessing the data will come from Spine Demographics? I think we should consider creating a single FHIR API spec for PDS, V&M and this - technically they are all just administrative resources queryable from a Spine Demographics API so not sure it makes sense to define three completely separate specs for this and risk making life unnecessarily hard for clients wanting to get this data (especially as it will be one codebase on Spine anyway) Thoughts?

**Comment by NaminderSoorma** [comment]
@agreenhalgh The V & M currently showing is out of date as a STU3 version has been done but needs to be promoted to the gh-pages branch. That is based on Care Connect API too, we have been told that we must follow the Care Connect API model for all work, we can remove un-wanted bits as I agree there is too much content on there.

@ahatherly I think it is also a good idea to combine work that is similar. I was also advised to create a "Medications" API instead of a Prescription centric one as there will most likely be more Medication related API's in the future. Maybe we need a meeting to discuss best way forward and Richard Kavanaugh will need to be involved as he determines our direction of travel in most cases.

**Comment by agreenhalgh** [comment]
@NaminderSoorma 
Agree there probably will be work in the meds domain, but these are demographics attributes (e.g. is the patient in receipt of pension credit) that happen to have a bearing on paying for prescriptions rather than anything to do with medications, so don't really belong in a 'medications' API.

**Comment by agreenhalgh** [comment]
@ahatherly yes theses statuses will be imported into Spine Demographics from their respective owners (NHS BSA, DWP...) and yes as above I agree these are demographic attributes. 

Is there anyone using V&M API such that redirecting those docs somewhere else would pull the rug under their feet?

**Comment by ahatherly** [comment]
I don't think so - there was only one supplier looking at it to my knowledge. It is a wierd one as there isn't actually a programme around it - it was directly commissioned into the DDC by DH. I'll ask DDC if they know any more about uptake. I did have an brief chat with Richard Kavanagh a while back and he was keen to combine V&M and PDS FHIR API specs (when we do the PDS one) so I doubt he'd object to also including this one.

**Comment by NaminderSoorma** [comment]
I have got agreement from Richard Kavanagh to rename this to a different name on @agreenhalgh request. However are we also looking to combine it with others into a bigger spec?
